benchmark/write.go

```diff
@@ -0,0 +1,56 @@
+// Writing files in Go follows similar patterns to the
+// ones we saw earlier for reading.
+
+package main
+
+import (
+	"math/rand"
+	"os"
+	"strconv"
+	"time"
+)
+
+const charset = "abcdefghijklmnopqrstuvwxyz" +
+	"ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789"
+
+var seededRand *rand.Rand = rand.New(
+	rand.NewSource(time.Now().UnixNano()))
+
+func main() {
+
+	f, err := os.OpenFile("data.sql", os.O_APPEND|os.O_WRONLY, 0600)
+
+	if err != nil {
+		panic(err)
+	}
+
+	defer f.Close()
+
+	for i := 1; i <= 100; i++ {
+		dataset := String(5)
+		f.WriteString("insert into DATASETS values (" + strconv.Itoa(i) + ", " + "\"" + dataset + "\"" + ");\n")
+		for j := 1; j <= 100; j++ {
+			block := String(4)
```

Comment:
block should have two pieces, first one is identical to dataset name and second unique hash id. Therefore I suggest to have:
```
blockHash := String(32) // here we may vary number of characters for block hash
block := fmt.Sprintf("%s#%s", dataset, blockHash)
```